protocol.html

```diff
@@ -559,6 +572,40 @@ <h3 property="schema:name">Resource Containment</h3>
                   <p id="server-hierarchical-containment">There is a 1-1 correspondence between containment triples and relative reference within the path name hierarchy. [<a href="https://github.com/solid/specification/issues/98#issuecomment-547506617" rel="cito:citesAsSourceDocument">Source</a>]. It follows that all resources are discoverable from a container and that it is not possible to create orphan resources. [<a href="https://github.com/solid/specification/issues/97#issuecomment-547459396" rel="cito:citesAsSourceDocument">Source</a>]</p>
 
                   <p><span about="" id="server-basic-container" rel="spec:requirement" resource="#server-basic-container"><span property="spec:statement">The representation and behaviour of containers in Solid corresponds to LDP Basic Container and <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> be supported by <span rel="spec:requirementSubject" resource="spec:Server">server</span>.</span></span> [<a href="https://github.com/solid/specification/issues/47#issuecomment-561675764" rel="cito:citesAsSourceDocument">Source</a>]</p>
+
+                  <section id="authoritative-resource-data" inlist="" rel="schema:hasPart" resource="#authoritative-resource-data">
+                    <h4 property="schema:name">Authoritative Resource Data</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>Container descriptions are not limited to containment triples. To further support client navigation and application interaction, servers can include <a href="#authoritative-information">authoritative information</a> about contained resources as part of the container description.</p>
+
+                      <p><span about="" id="authoritative-contained-resource-data" rel="spec:requirement" resource="authoritative-contained-resource-data"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> include the following information about contained resources as part of the container description, unless that information does not exist or is expensive to determine:</span></span></p>
+
+                      <dl>
+                        <dt><code>rdf:type</code></dt>
+                        <dd>A class whose URI formed by concatenating <code>http://www.w3.org/ns/iana/media-types/</code>, the IANA Media Type [<cite><a class="bibref" href="#bib-iana-media-types">IANA-MEDIA-TYPES</a></cite>] of the resource, and <code>#Resource</code>.</dd>
```

Comment:
We need to mention that no escaping is performed, so the slash remains as-is.

Reply:
I've updated to be more specific about the URI that's being constructed.. which is actually a URI Template:

https://github.com/solid/specification/pull/352/commits/5ea32a4e440eb952b9e11ec0ec97cc27c4ca843e

I thought the current language would imply that no encoding transformation is applied on the value, but if you think that it'd be better to use something like:

"..expands to a value <ins>without an encoding transformation</ins> that is.."

we can do that.